jobs/base_model_validator_errors_test.py

```diff
@@ -34,8 +34,14 @@ class MockModel(base_models.BaseModel):
     pass
 
 
+class BaseCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
+    """Base class for commit model validator tests """
+
+    pass
```

Comment:
Ah sorry for being unclear, I meant let's use `base_models.BaseCommitLogEntryModel` everywhere and not introduce any subclasses at all.

Reply:
Done